README.md

```diff
@@ -1 +1,198 @@
+
+```xml
+<gazebo>
+  <plugin filename="libgazebo_ros2_control.so" name="gazebo_ros2_control">
+    <ros>
+      <namespace>/</namespace>
+      <parameter name="slider_to_cart.p" type="double">50.0</parameter>
+      <parameter name="slider_to_cart.i" type="double">10.0</parameter>
+      <parameter name="slider_to_cart.d" type="double">15.0</parameter>
+      <parameter name="slider_to_cart.i_clamp_max" type="double">3.0</parameter>
+      <parameter name="slider_to_cart.i_clamp_min" type="double">-3.0</parameter>
+      <parameter name="slider_to_cart.antiwindup" type="bool">false</parameter>
+    </ros>
+    ...
+  </plugins>
+</gazebo>
+```
+
+#### Executing the examples
+
+There are some examples in the `gazebo_ros2_control_demos` package. These examples allow to launch a cart in a 30 meter rail.
+
+![](img/cart.gif)
+
+You can run some of the configuration running the following commands:
+
+```bash
+ros2 launch gazebo_ros2_control_demos cart_example_position_pid.launch.py
+ros2 launch gazebo_ros2_control_demos cart_example_position.launch.py
+ros2 launch gazebo_ros2_control_demos cart_example_velocity.launch.py
+```
+
+Send example commands:
+
+When the Gazebo world is launched you can run some of the following commads to move the cart.
+
+```bash
+ros2 run gazebo_ros2_control_demos example_position
+ros2 run gazebo_ros2_control_demos example_velocity
+```
+
+To get or modify the values of the PID controller you can run the following commands:
+
+```bash
+ros2 param get /gazebo_ros2_control slider_to_cart.p
+ros2 param get /gazebo_ros2_control slider_to_cart.i
+ros2 param get /gazebo_ros2_control slider_to_cart.d
+ros2 param get /gazebo_ros2_control slider_to_cart.i_clamp_max
+ros2 param get /gazebo_ros2_control slider_to_cart.i_clamp_min
+```
+
+```bash
+ros2 param set /gazebo_ros2_control slider_to_cart.p 50.0
+ros2 param set /gazebo_ros2_control slider_to_cart.i 10.0
+ros2 param set /gazebo_ros2_control slider_to_cart.d 15.0
+ros2 param set /gazebo_ros2_control slider_to_cart.i_clamp_max 3.0
+ros2 param set /gazebo_ros2_control slider_to_cart.i_clamp_min -3.0
+```
+
+#### Dependencies
+
+ - gazebo_ros_pkgs
+ - realtime_tools
+ - ros2_control
+ - ros2_controllers
+ - xacro
+
+Step to compile `gazebo_ros2_control`
+
+```bash
+mkdir -p ~/ros2_control_ws/src
+cd ~/ros2_control_ws/src
+git clone https://github.com/ros-simulation/gazebo_ros_pkgs -b ros2
+git clone https://github.com/ros-controls/realtime_tools -b ros2_devel
+git clone https://github.com/ros-controls/ros2_control
+git clone https://github.com/ros-controls/ros2_controllers
+git clone https://github.com/ddengster/ros2_control/ -b coffeebot_deps ros2_control_ddengster
+touch ros2_control_ddengster/COLCON_IGNORE
+cp -r ros2_control_ddengster/joint_limits_interface ros2_control_ddengster/transmission_interface  ros2_control
```

Comment:
How about adding a comment linking to these issues here, so the reader can be aware of why they're cloning this unofficial package and can track when it can be substituted:

* https://github.com/ros-controls/ros2_control/issues/26
* https://github.com/ros-controls/ros2_control/issues/32

Reply:
ping @ahcorde

Reply:
I removed this part of the tutorial in favour of the Dockerfile. But I included a note at the end of the README.md https://github.com/ros-simulation/gazebo_ros2_control/pull/12/commits/cbb2903db37e6b29348ece89cd47086e5ed4ca59